### PR TITLE
#8387 Spaces/User admin list edit quota batch actions - Followup

### PR DESCRIPTION
**Pull request by lookacat**
## Description
See https://github.com/owncloud/web/pull/8387

## Issue
Fixes https://github.com/owncloud/web/issues/8417

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-pkg/src/components/Spaces/QuotaModal.vue [hunk 1]
```diff
@@ -69,7 +69,11 @@ export default defineComponent({
     }
   },
   mounted() {
-    this.selectedOption = this.spaces[0].spaceQuota.total || 0
+    if (!this.spaces.length) {
+      this.selectedOption = 0
+      return
+    }
+    this.selectedOption = this.spaces[0].spaceQuota?.total || 0
```
Comment (kulmann) [inline packages/web-pkg/src/components/Spaces/QuotaModal.vue:76]:
```suggestion
    this.selectedOption = this.spaces[0]?.spaceQuota?.total || 0
```
With this the early return above is not needed. `this.spaces[0]` is still valid when the array is empty.

File: packages/web-app-admin-settings/src/views/Spaces.vue [hunk 2]
```diff
@@ -135,7 +135,7 @@ export default defineComponent({
     const { sideBarOpen, sideBarActivePanel } = useSideBar()
 
     const loadResourcesEventToken = ref(null)
-    const updateQuotaForSpaceEventToken = ref(null)
+    const updateQuotaForSpaceEventToken = null
```
Comment (kulmann) [inline packages/web-app-admin-settings/src/views/Spaces.vue:138]: Needs to stay `ref(null)`. Why did you change it? 🤔

File: packages/web-app-admin-settings/src/views/Users.vue [hunk 3]
```diff
@@ -219,11 +219,12 @@ export default defineComponent({
...
-      const requests = []
-      unref(selectedUsers).forEach((user) => {
-        requests.push(loadAdditionalUserDataTask.perform(user))
-      })
-      await Promise.all(requests)
+      const requests = unref(selectedUsers).map((user) => loadAdditionalUserDataTask.perform(user))
+      const results = await Promise.allSettled<Array<unknown>>(requests)
+      const failedRequests = results.filter((result) => result.status === 'rejected')
+      if (failedRequests.length > 0) {
+        console.error('Failed to load additional user data', failedRequests)
```
Comment (kulmann) [inline packages/web-app-admin-settings/src/views/Users.vue:226]: This will only fail for users who never logged in, right? Since that's expected, shouldn't we avoid the console noise? Maybe reducing the log level to `debug` instead of `error` is an option

File: packages/web-app-admin-settings/src/views/Spaces.vue (hunk 2 above)
Comment (JammingBen) [reply packages/web-app-admin-settings/src/views/Spaces.vue:138]: I think that refers to a comment I made in the last PR, where I said that this does not need to be a `ref` because it's not reactive. Although you need to make it a `let` and remove `.value` (and/or `unref()`) down below.

Comment (kulmann) [reply packages/web-app-admin-settings/src/views/Spaces.vue:138]: Ah ok. Yes, simply removing the wrapping `ref()` is not sufficient. That's why I was confused.

File: packages/web-app-admin-settings/src/views/Users.vue (hunk 3 above)
Comment (lookacat) [reply packages/web-app-admin-settings/src/views/Users.vue:226]: sounds good, just felt bad to not have any error handling

**Review by fschade — APPROVED** [review]
one last thing, please update the related changelog to contain this pr too.
Everything else, LGTM

**Review by kulmann — APPROVED** [review]
👍

**Comment by kulmann** [comment]
> one last thing, please update the related changelog to contain this pr too.
> Everything else, LGTM

From what I see that was already the case :-) If I overlooked something please followup again
